manifests/plugin/disk.pp

```diff
@@ -15,7 +15,7 @@
   validate_bool($ignoreselected)
 
   if $::osfamily == 'RedHat' {
-    if $manage_package {
+    if $manage_package != undef {
```

Comment:
Curious... Maybe it's dependent on the version of puppet?

```
alex@alex-VirtualBox:~$ cat test.pp
if 'false' {
  notice("'false' is true")
}
if false {
  notice("false is true??")
} else {
  notice("but false is false!")
}
alex@alex-VirtualBox:~$ puppet apply test.pp
Notice: Scope(Class[main]): 'false' is true
Notice: Scope(Class[main]): but false is false!
Notice: Compiled catalog for alex-virtualbox.ordsvy.gov.uk in environment production in 0.07 seconds
Notice: Applied catalog in 0.01 seconds
alex@alex-VirtualBox:~$ puppet --version
4.5.0
```

Anyway, congrats on adding a spec test!  I suppose you could add a context with `:manage_package => 'false'`  Hopefully watch it fail (on some versions of puppet only perhaps).  Then add a validate_bool and change the expectation to something similar to the test in `context ':disks is not an array'`

But only if you're bored and want to do some playing around with rspec-puppet! ;) If you're short of time, I'm happy with merging without the extra tests.